userService/src/main/resources/application.yml

```diff
@@ -0,0 +1,12 @@
+#spring:
```

Comment:
There should be no code that is not being used. Why is this commentedv?

Reply:
this has been fixed